mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
Or to put it differently, as in the original description:

> As shown in the code, compute_raw_covariance has (effectively) keep_sample_mean=False by default, whereas compute_covariance has keep_sample_mean=True by default. This makes me think we should have keep_sample_mean=None -> False with deprecation message saying it will change to True in 0.13. This would really unify the two functions. Thoughts?